platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -195,10 +242,10 @@ private static CloseableHttpClient getHttpClient(boolean retryRequestsOnError) {
     }
 
     private static SSLContext getSslContext() {
-        if (System.getProperty("javax.net.ssl.keyStore") == null ||
-                System.getProperty("javax.net.ssl.keyStorePassword") == null ||
-                System.getProperty("javax.net.ssl.trustStore") == null ||
-                System.getProperty("javax.net.ssl.trustStorePassword") == null) {
+        if (System.getProperty("javax.net.ssl.keyStore") == null || System.getProperty(
+                "javax.net.ssl.keyStorePassword") == null || System.getProperty(
+                "javax.net.ssl.trustStore") == null || System.getProperty(
+                "javax.net.ssl.trustStorePassword") == null) {
```

Comment:
Did the formatter format this this way? Maybe it's just a preference thing, but I think this was more readable beforehand.